src/test/java/io/rtr/conduit/amqp/impl/AMQPAsyncConsumerBuilderTest.java

```diff
@@ -39,6 +39,49 @@ public void testValidationQueueRequiredWhenNotDynamic(){
     }
 
     @Test
+    public void testValidationBasicConfig(){
+        AMQPAsyncConsumerBuilder amqpAsyncConsumerBuilder = AMQPAsyncConsumerBuilder.builder()
+                .ensureBasicConfig("exchange", AMQPConsumerBuilder.ExchangeType.DIRECT, "queue", "routingKey");
+        AMQPCommonListenProperties commonListenProperties = amqpAsyncConsumerBuilder.buildListenProperties();
+
+        assertEquals("Can not ensureBasicConfig and be dynamic", false, commonListenProperties.isDynamicQueueCreation());
```

Comment:
not sure what you gain from this assertion, the property isn't set and you already have a test for the illegal arg that is thrown with this combination.